### PR TITLE
Add Vincent

### PR DESCRIPTION
**Pull request by VAOkolo**
making first PR after adding username and name to 'roster.json' for Morgan cohort as part of pre-work

### CODE REVIEW

File: morgan/roster.json [hunk 1]
```diff
@@ -41,6 +41,9 @@
     {
       "name": "Muhammed Ismael Ali",
       "github": "Ismael76"
+    },
+    {
+      "name": "Vincent Okolo",
+      "Github": "VAOkolo"
```
Comment (rom-30) [inline morgan/roster.json:47]: Can you please write github in lowercase? you are also missing the closing square bracket of the students array :)

**Review by rom-30 — CHANGES_REQUESTED** [review]

File: morgan/roster.json (hunk 1 above)
Comment (VAOkolo) [reply morgan/roster.json:47]: Will adjust and give it another go. Thanks mate :)

Comment (VAOkolo) [reply morgan/roster.json]: should be ready to merge in

**Review by rom-30 — APPROVED** [review]
Thanks @VAOkolo

**Pull request merged**